panels/UserPanel.php

```diff
@@ -198,33 +198,37 @@ public function save()
         $identity = Yii::$app->user->identity;
 
         if (!isset($identity)) {
-            return ;
+            return;
         }
 
-        $authManager = Yii::$app->getAuthManager();
-
         $rolesProvider = null;
         $permissionsProvider = null;
 
-        if ($authManager instanceof yii\rbac\ManagerInterface) {
-            $roles = ArrayHelper::toArray($authManager->getRolesByUser(Yii::$app->getUser()->id));
-            foreach ($roles as &$role) {
-                $role['data'] = $this->dataToString($role['data']);
-            }
-            unset($role);
-            $rolesProvider = new ArrayDataProvider([
-                'allModels' => $roles,
-            ]);
-
-            $permissions = ArrayHelper::toArray($authManager->getPermissionsByUser(Yii::$app->getUser()->id));
-            foreach ($permissions as &$permission) {
-                $permission['data'] = $this->dataToString($permission['data']);
-            }
-            unset($permission);
+        try {
```

Comment:
using a Exception catcher is quite rigid. Can't it be closer related to the issue at hand? Shouldnt we fix this within DbManager or PhpManager to make these throw notifications about Caching instead of exceptions?